fling/utils/torch_utils.py

```diff
@@ -43,6 +43,7 @@ def seed_everything(seed):
     random.seed(seed)
     if torch.cuda.is_available():
         torch.cuda.manual_seed(seed)
+    torch.backends.cudnn.deterministic = True
```

Comment:
This might largely slow down the computation speed. I think it should be removed.